### ISSUE TITLE
input_boolean helper UI: initial state field not displayed.

### ISSUE THREAD

**Issue by oncleben31** [home-assistant/frontend#5550]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

Similar to #5540, when I try to add an input_boolean thru the Helpers menu. The field for setting the potential initial value is missing.

![image](https://user-images.githubusercontent.com/93244/79381274-0542ae00-7f62-11ea-90e0-bdc52fdcc900.png)

## Expected behavior

We should have access to the field to set the initial value as we can do in YAML configuration mode.

## Steps to reproduce

UI -> Helpers -> Add input boolean

## Environment

- Home Assistant release with the issue: 0.108.4
- Last working Home Assistant release (if known): don't know
- Browser and browser version: Chrome 80.0.3987.163 and Safari 13.0.5
- Operating system: MacOS 10.15.3

## Problem-relevant configuration

N/A

## Javascript errors shown in your browser console/inspector

None

**Comment by oncleben31** [comment]
This is not a bug ? This not implemented ? Strange as I have the text explaining what should be the field not displayed.
Strange with the UI in English the text is not displayed.

**Comment by ludeeus** [comment]
Initial state is currently not implemented for that helper type.
https://github.com/home-assistant/frontend/blob/dev/src/panels/config/helpers/forms/ha-input_boolean-form.ts#L55-L83

**Comment by bramkragten** [comment]
Yeah that text should be removed. Initially, it was implemented but we decided to remove it.

**Comment by patmann03** [comment]
Definitely interested in having the ability to set initial state.  I created an input_number and am stuck with the default value.  For the time being, I've set an automation that will set the value.  Definitely like having the ability to create these from the front end vs. yaml!

**Comment by bramkragten** [comment]
Just curious, what do you use that for?

**Comment by cmitz** [comment]
I use helpers a lot to disable some automations, most automations are enabled, but with these examples I want to disable them temporarily when something is wrong, when on vacation, on when there's another edgecase:
- Automatically set heating to 'off' when I'm not home – sometimes my "is home" sensor malfunctions
- Disable the motion enabled lights turning off automatically – when I'm working on something small so not moving much
- Disable notifications my backup is too old – "I KNOW, I'm working on it!!"

I think you get the point ;)

I also use input.date_time helpers for tweaking the time I want to wake up, go to sleep, get notifications what my day tomorrow will look like etc and whether those should work on the weekends, too

TL;DR: My helpers aren't updated often, but setting them after every HA restart is cumbersome

**Comment by bwarman** [comment]
Is this ever going to be implemented? I'm in the same situation that cmitz is where I need an input boolean to be true initially, because it means my default automations will run. I'm specifically making one to turn on/off Alexa announcements. Most of the time I want them on but sometimes I don't. Logically I would create and input boolean for "Announcements On" and it would default to "On". For those other times I could just turn them "off".

**Comment by bramkragten** [comment]
We removed this as it probably is confusing and most people will want the value restored from before the restart.

**Comment by bwarman** [comment]
So the input_boolean will return to its last known state upon restart?

**Comment by bramkragten** [comment]
Yes

**Comment by cmitz** [comment]
> Yes

I did not know that and that would look like it indeed solves the use-case. I will check it tonight and report back if I'm still missing something.

**Comment by cmitz** [comment]
I can confirm my use-case was fixed since the values are persisted through reboots. Very nice!
